README.md

```diff
@@ -61,7 +61,15 @@ Run the command to build the code `mvn clean install`
 
 If all goes well, the build should be successful and a war should be produced in `web/web.rya/target/web.rya.war`
 
-Note: To perform a build of the geomesa/lucene indexing, run the build with the profile 'indexing' `mvn clean install -P indexing`
+Note: The following profiles are available to tailor the build:
+
+
+| Profile ID | Purpose |
+| ---------- | ------- |
+| geoindexing | perform a build of the geomesa/lucene indexing |
+| mongodb | build with mongoDB configuration (defaults to accumulo) |
+
+To run the build with the profile 'geoindexing' `mvn clean install -P geoindexing`.
```

Comment:
was this for another jira ticket.  I'm confused over why it was bundled here or if this is just a bad rebase or something